### PR TITLE
11.0 mig pos default payment method

### PR DESCRIPTION
**Pull request by CGenie**

### CODE REVIEW

File: pos_default_payment_method/tests/test_constrain.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
```
Comment (chienandalu) [inline pos_default_payment_method/tests/test_constrain.py]: Remove all utf-8 shebangs

File: pos_default_payment_method/README.rst [hunk 2]
```diff
@@ -0,0 +1,64 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```
Comment (chienandalu) [inline pos_default_payment_method/README.rst:1]: Not blocking, but recommended to use readme fragmentes (https://github.com/OCA/maintainer-tools/tree/master/template/module/readme) as it makes the readme more maintainable through versions.

**Review by chienandalu — COMMENTED** [review]
Looks ok overall, anyway this branch seems to be still broken by pos_loyalty tests :confused:

**Comment by pedrobaeza** [comment]
Please squash together adjacent "OCA Transbot.." commits using https://github.com/OCA/maintainer-tools/wiki/Merge-commits-in-pull-requests

and check Travis status

File: pos_default_payment_method/tests/test_constrain.py (hunk 1 above)
Comment (CGenie) [reply pos_default_payment_method/tests/test_constrain.py]: OK done.

**Comment by CGenie** [comment]
I squashed the commits. However, it's `pos_loyalty` that is failing, as @chienandalu noted.

**Review by legalsylvain — APPROVED** [review]
Code review / no test.

**Comment by legalsylvain** [comment]
All is now green. 

Merging PR.

/ocabot merge patch

**Pull request merged**